README.md

```diff
@@ -15,7 +15,17 @@ it can currently deploy stateful apps as well, but there is no UI feature to int
 
 ### tldr
 
-clone this repo, go into the `src/server` directory, then `pip install -r requirements.txt`. you can locally start your server by `flask run`, and you'll see the editor on localhost port 5000. in order to compile and deploy apps, you need to have [sandbox](https://github.com/algorand/sandbox) running. 
+Make sure you have [sandbox](https://github.com/algorand/sandbox) running. 

```

Comment:
thanks for cleaning this up 😄  I think you can delete the `### tldr` above this line now